elements/pfelement/src/pfelement.js

```diff
@@ -39,8 +39,8 @@ class PFElement extends HTMLElement {
     return this.querySelector(`[slot='${name}']`);
   }
 
-  has_slot(name) {
-    return this.querySelector(`[slot='${name}']`);
+  has_slots(name) {
```

Comment:
@markcaron I checked the entire codebase and didn't see any instances of us using `has_slots`. While I do see this being useful, if we're not using it anywhere, I think we should just remove the function.

Reply:
@kylebuch8 pfe-band is using `has_slot` singular and I like the idea of the two being separate, `has_slot` versus `has_slots` so you can tell by the name of the function what return to expect. 👍  It sounds like @markcaron needs the `has_slots` in a soon-to-be-built component so I vote yes to this update.